### PR TITLE
Bump version of sqlx, sea-query, sea-schema, time and uuid

### PR DESCRIPTION
**Pull request by billy1624**
## PR Info

- Closes https://github.com/SeaQL/sea-orm/issues/808

## Breaking Changes

- [x] Upgrade `time` to ^0.3
- [x] Upgrade `sqlx` to ^0.6
- [x] Upgrade `uuid` to ^1
- [x] Upgrade `sea-query` to ^0.26
- [x] Upgrade `sea-schema` to ^0.9.2
- [x] https://github.com/launchbadge/sqlx/issues/1618  
  SQLite `chrono::DateTime<FixedOffset>` timezone fix
    * `DateTime<FixedOffset>` will be stored in SQLite with the correct timezone instead of always in UTC.
      This was flagged as a "potentially breaking change" since it changes how dates are sent to SQLite.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -25,19 +25,19 @@ path = "src/lib.rs"
...
-time = { version = "^0.2", optional = true }
+time = { version = "^0.3", optional = true }
 futures = { version = "^0.3" }
 futures-util = { version = "^0.3" }
 log = { version = "^0" }
 tracing = { version = "0.1", features = ["log"] }
 rust_decimal = { version = "^1", optional = true }
 sea-orm-macros = { version = "^0.8.0", path = "sea-orm-macros", optional = true }
-sea-query = { version = "^0.24.5", features = ["thread-safe"] }
+sea-query = { version = "^0.26.0", features = ["thread-safe"] }
```
Comment (ikrivosheev) [inline Cargo.toml:35]: Maybe without pin minor version?

File: Cargo.toml [hunk 2]
```diff
@@ -53,6 +53,7 @@ rust_decimal_macros = { version = "^1" }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 sea-orm = { path = ".", features = ["mock", "debug-print"] }
 pretty_assertions = { version = "^0.7" }
+time = { version = "^0.3", features = ["macros"] }
```
Comment (ikrivosheev) [inline Cargo.toml:56]: Maybe merge with privies time?

**Review by ikrivosheev — COMMENTED** [review]
@billy1624 thank you, small comments.

File: Cargo.toml (hunk 2 above)
Comment (billy1624) [reply Cargo.toml:56]: I don't want `macros` feature to be enabled by default. I just need it for `sea-orm` tests

Comment (ikrivosheev) [reply Cargo.toml:56]: Ah, sorry, these are different files...

File: Cargo.toml (hunk 1 above)
Comment (billy1624) [reply Cargo.toml:35]: Do you mean the patch version?
- `<major>.<minor>.<patch>`

Yeah, we can relax that

**Review by ikrivosheev — APPROVED** [review]
LGTM!

File: Cargo.toml (hunk 2 above)
Comment (billy1624) [reply Cargo.toml:56]: NVM :D

**Review by tyt2y3 — APPROVED** [review]
Awesome! Thank you all

**Pull request merged**